staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -102,6 +102,12 @@ service RuntimeService {
     // ListContainerStats returns stats of all running containers.
     rpc ListContainerStats(ListContainerStatsRequest) returns (ListContainerStatsResponse) {}
 
+    // PodSandboxStats returns stats of the pod. If the pod does not
+    // exist, the call returns an error.
+    rpc PodSandboxStats(PodSandboxStatsRequest) returns (PodSandboxStatsResponse) {}
+    // ListPodSandboxStats returns stats of all running pods.
+    rpc ListPodSandboxStats(ListPodSandboxStatsRequest) returns (ListPodSandboxStatsResponse) {}
+
```

Comment:
What is the reason to add it to `v1alpha2`, too?

Reply:
because containerd hasn't switched to v1 yet and tests failed here without adding for v1alpha2 as well. I'd be fine dropping if the tests end up passing